platform/build.gradle

```diff
@@ -16,3 +16,9 @@ dependencies {
 moduleJvmArgs {
     opens 'java.base/java.lang'
 }
+/**
+ * Needs to be kept at Java 11 to keep compatibility with <a href="https://github.com/palantir/gradle-jdks/blob/develop/gradle-jdks-setup/build.gradle#L29">gradle-jdks-setup</a>
```

Comment:
Whenever you link, do a link to a particular commit rather than a branch like develop that will immediately get out of date. Just press `y` on github to make a link into a commit-based link.